src/tools/terraform.ts

```diff
@@ -0,0 +1,65 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import * as os from 'os'
+import * as unzipper from 'unzipper'
```

Comment:
change back to the in-function `require` statement. We will revisit the unzipper down the road as we also need to add the type definition dependency